python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -218,6 +220,44 @@ def get_metadata(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, Any
         """
         return default_metadata_from_dbt_resource_props(dbt_resource_props)
 
+    @classmethod
+    @public
+    def get_tags(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, str]:
+        """A function that takes a dictionary representing properties of a dbt resource, and
```

Comment:
Should also mention that invalid tags will be ignored by default. Adding a warning log might be too aggro, but the behavior should at least be documented.